client/infinispan-key-value-store-hotrod/src/main/java/org/infinispan/api/reactive/client/impl/ContinuousQueryPublisherImpl.java

```diff
@@ -0,0 +1,86 @@
+package org.infinispan.api.reactive.client.impl;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.infinispan.api.reactive.EntryStatus;
+import org.infinispan.api.reactive.KeyValueEntry;
+import org.infinispan.client.hotrod.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.query.api.continuous.ContinuousQuery;
+import org.infinispan.query.api.continuous.ContinuousQueryListener;
+import org.infinispan.query.dsl.Query;
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+
+import io.reactivex.Flowable;
+import io.reactivex.processors.UnicastProcessor;
+
+/**
+ * Implements the {@link Publisher<KeyValueEntry<K, V>>} using RXJava and Flowable API.
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 10.0
+ */
+public class ContinuousQueryPublisherImpl<K, V> implements Publisher<KeyValueEntry<K, V>> {
+   private static final Log log = LogFactory.getLog(ContinuousQueryPublisherImpl.class, Log.class);
+
+   private final ContinuousQuery<K, V> continuousQuery;
+   private final boolean created;
+   private final boolean updated;
+   private final boolean deleted;
+   private final Query query;
+   private Flowable flowable;
+   private Set<Subscriber<? super KeyValueEntry<K, V>>> subscribers = ConcurrentHashMap.newKeySet();
+   private ContinuousQueryListener continuousQueryListener;
```

Comment:
can you add the missing types to `ContinuousQueryListener` and `UnicastProcessor`?
the amount of warnings in IntelliJ is annoying xD

Reply:
ok